enterprise/backend/test/metabase_enterprise/sso/integrations/saml_test.clj

```diff
@@ -460,7 +460,10 @@
                           (map #(dissoc % :last_login)))))
               (testing "attributes"
                 (is (= (some-saml-attributes "newuser")
-                       (saml-login-attributes "newuser@metabase.com")))))
+                       (saml-login-attributes "newuser@metabase.com"))))
+              (testing "User Invite Event is logged."
+                (is (= "newuser@metabase.com"
+                       (get-in (t2/select-one :model/AuditLog :topic :user-invited) [:details :email])))))
```

Comment:
added this inside the test to piggyback on the setup/teardown so that it's not doubled up for this single check.